be/src/util/threadpool.cpp

```diff
@@ -444,6 +451,15 @@ bool ThreadPool::wait_for(const MonoDelta& delta) {
                                [&]() { return _total_queued_tasks <= 0 && _active_threads <= 0; });
 }
 
+void ThreadPool::update_max_threads(int max_threads) {
+    if (max_threads < this->_min_threads) {
+        LOG(WARNING) << "invalid max threads num : " << max_threads << " min threads num: " << this->_min_threads;
```

Comment:
If set failed, should an error be returned?

Reply:
Ok, Done